libmscore/textbase.cpp

```diff
@@ -519,8 +519,35 @@ void TextFragment::draw(QPainter* p, const TextBase* t) const
       {
       QFont f(font(t));
       f.setPointSizeF(f.pointSizeF() * MScore::pixelRatio);
+#if defined Q_OS_WIN || defined Q_OS_LINUX
```

Comment:
Not BSD? Everything but MAC?

Reply:
In the issue report I think only Windows and Linux example were posted.
Under Mac I didn't see the bug and I couldn't find Mac user complaining about such a bug, since the font engine is not freetype in Mac.
This bug is given by a specific interplay of Qt and freetype.
Do you think I should use something like "if not defined Q_OS_MAC"?

Reply:
Probably. BSD and Solaris and other Unicēs also use Freetype.

Reply:
Good you chipped in yourself voluntary, else I'd have dragged you in om this particular one ;-)

Reply:
Why not use `#ifdef`?

Reply:
Because it is aways done that way when `||`ing 2 such defines

Reply:
Yeah you're right. I thought this was unfamiliar because normally I see this:
`#if (defined (_MSCVER) || defined (_MSC_VER))`
So I think it's probably better to have parentheses.